terraform-plans/templates/github/charm_release.yaml.tftpl

```diff
@@ -10,6 +10,7 @@ on:
     branches: [ main ]
   release:
     types: [ published ]
+  workflow_dispatch:
```

Comment:
@gabrielcocenza actually can we add these in a separate PR?  They aren't related to adding charmed-openstack-exporter-snap.